src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -63,6 +66,33 @@ export const PhoneNumberInput: FunctionComponent<{
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   onSubmit = () => {}
 }) => {
+  const formatPhoneNumber = (
+    mobileNumberValue: string,
+    region = "SG"
+  ): string => {
+    try {
+      return phoneUtil.format(
+        phoneUtil.parse(mobileNumberValue, region),
+        PhoneNumberFormat.NATIONAL
+      );
+    } catch (e) {
+      if (mobileNumber && mobileNumber?.length < mobileNumberValue.length) {
+        return mobileNumber;
+      }
+      return mobileNumberValue;
+    }
+  };
+  const [mobileNumber, setMobileNumber] = useState<string>(
+    formatPhoneNumber(mobileNumberValue.replace(" ", ""))
+  );
+
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const onHandleChangePhoneNumber = (text: string) => {
```

Comment:
I would try to avoid disabling `eslint` rules and only really disable them if you cannot easily fix it:
```suggestion
  const onHandleChangePhoneNumber = (text: string): void => {
```